TODO.md

```diff
@@ -43,12 +43,10 @@ The following files are yet to be converted:
 - src/scrot_selection.c
 - src/selection_classic.c
 - src/selection_edge.c
-- src/slist.c
```

Comment:
names are not deleted, [please see here]( https://github.com/resurrecting-open-source-projects/scrot/pull/121/files#diff-5c6a1301c6b59b30a040d747d065e861d3dd98bde0e5a4356d92d594e9835986R54)

Reply:
What do you mean by 'names' here?